vmdb/app/controllers/api_controller/initializer.rb

```diff
@@ -36,6 +36,17 @@ def collection_config
         @config[:collections]
       end
 
+      #
+      # Let's fetch encrypted attribute names of objects being rendered if not already done
+      #
+      def fetch_encrypted_attribute_names(obj)
+        @encrypted_objects_checked ||= {}
+        klass = obj.class.name
+        return unless @encrypted_objects_checked[klass].nil?
+        @encrypted_objects_checked[klass] = object_encrypted_attributes(obj)
+        @encrypted_objects_checked[klass].each { |attr| @attr_encrypted[attr] = true }
+      end
```

Comment:
Why is this method needed...it seems to check for encrypted columns and then caches it, but you already cache that in `@attr_encrypted` on line 118.